lib/coinatmradar/coinatmradar.js

```diff
@@ -133,7 +133,7 @@ function sendRadar (data) {
 function mapRecord (rates, settings) {
   const timestamp = new Date().toISOString()
   return Promise.all([getMachines(rates, settings), getOperatorId('coinatmradar')])
-    .then(([machines, operatorId]) => {
+    .then(([machines, { operatorId }]) => {
```

Comment:
Can't you do this on the "getOperatorId" function? it's unexpected to ask for a ID and receive a object in response.

Reply:
Also do take a look at other places using this function to see if they are using it correctly